core/examples/luigi-sample-angular/src/assets/extendedConfiguration.js

```diff
@@ -434,13 +466,13 @@ Luigi.setConfig({
           label: 'Settings',
           viewUrl: '/sampleapp.html#/settings'
         },
-        {
-          label: 'Open Google in this tab',
-          externalLink: {
-            url: 'http://google.com',
-            sameWindow: true
-          }
-        },
+        // {
```

Comment:
Is there any reason while we commented this code?

Reply:
Just to save space in the top navi, I am removing the code completely.